pkg/analytics/instrumentation_collector_test.go

```diff
@@ -75,6 +35,10 @@ func Test_InstrumentationCollector(t *testing.T) {
 	})
 
 	t.Run("it sets the userAgent application data", func(t *testing.T) {
+		ic := setupBaseCollector(t)
+		expectedV2InstrumentationObject := buildExpectedBaseObject(t)
+
+		mockUserAgent := networking.UserAgentInfo{}
```

Comment:
These were added as previously, as the tests were dependent on each other. Now they're added explicitly.